src/Content/Backend/Solution/.template.config/template.json

```diff
@@ -54,6 +54,13 @@
 			"displayName": "Disable Authentication/Authorization",
 			"description": "Disable Authentication and Authorization implementation for the solution"
 		},
+		"workerService": {
```

Comment:
I think we also need to update the `ide.host.json` file so that this option is available when creating new projects using the IDE.